### ISSUE TITLE
Week 3 - remove Jake Porway

### ISSUE THREAD

**Issue by nikzei** [GovLab/academy-courses#18]
Early next week, after I get confirmation from Matt, replace with <a href="http://www.localdata.com">Matt Hampel</a>.

**Comment by luisdaniel** [comment]
any confirmation from Matt?

**Comment by luisdaniel** [comment]
still waiting on confirmation from Matt?

**Comment by luisdaniel** [comment]
removed Jake and closing issue. If you get confirmation from Matt, you can reopen the issue.

**Comment by nikzei** [comment]
Matt is on board!
And we shot a video of him that's forthcoming.

**Comment by luisdaniel** [comment]
updated. submit new issue with details when you have video
